jaeger-dropwizard/src/main/java/com/uber/jaeger/dropwizard/Configuration.java

```diff
@@ -39,9 +41,15 @@ public Configuration(
   }
 
   @Override
-  public synchronized Tracer getTracer() {
+  public synchronized com.uber.jaeger.Tracer getTracer() {
     if (disable) {
-      return NoopTracerFactory.create();
+      Metrics metrics = new Metrics(statsFactory);
+      ConstSampler sampler = new ConstSampler(false);
+      Reporter reporter = new NoopReporter();
+      return new com.uber.jaeger.Tracer.Builder(serviceName, reporter, sampler)
```

Comment:
@yurishkuro should we store a reference to this tracer (maybe call it no-op tracer), so that a new one does not have to be built on every call?

Reply:
yes, I think it should be stored the same way as if it was the regular tracer.